Cargo.toml

```diff
@@ -76,7 +76,8 @@ license = "Apache-2.0"
 repository = "https://github.com/risingwavelabs/risingwave"
 
 [workspace.dependencies]
-foyer = { version = "0.8.6", features = ["nightly"] }
+# foyer = { version = "0.9", features = ["nightly"] }
```

Comment:
I think we will wait for the foyer new release before merging this PR?

Reply:
Yep. I'm waiting for this PR to be approved. If there is any need to modify foyer, I can fix it and release a new version.